gui/system/migrations/0029_adv_boot_scrub.py

```diff
@@ -0,0 +1,16 @@
+from django.db import migrations, models
```

Comment:
I believe we can remove that migration, it does not introduce any actual database change and will cause conflicts with master branch.